### DISCUSSION TITLE
deepin-clone: exclude function?

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#2976] [Features Request & Ideas | 特性请求 & 头脑风暴]
is there have ```exclude``` function?

### why need?
* my ```/dev/sdb3``` mounted at ```/userdata```
* this disk neednt backup

### bug?
* cant set backup location at ```/userdata```
* all of ```/userdata``` will be backuped when do a full system backup

system: uos v20 1011    
deepin-clone 5.0.3

thanks!

**Comment by meiyixiang** [comment]
We plan to implement this function in the future, but it may be realized in combination with the whole backup and restoration function and partition mode. It is under development, but the cycle will be long, please wait patiently
